handler/src/main/java/io/netty/handler/ssl/OpenSsl.java

```diff
@@ -198,6 +230,14 @@ public static Throwable unavailabilityCause() {
     }
 
     /**
+     * Returns all the available cipher suites (Java-style).
+     * Please note that the returned array may include the cipher suites that are insecure or non-functional.
+     */
+    public static Set<String> availableCipherSuitesConverted() {
```

Comment:
`availableJavaCipherSuites`?

should we consider renaming `availableCipherSuites` to `availableOpenSslCipherSuites` to future clarify (interface breaking change).